lighty-examples/lighty-community-restconf-netconf-app/pom.xml

```diff
@@ -32,10 +32,6 @@
             <groupId>io.lighty.modules</groupId>
             <artifactId>lighty-netconf-sb</artifactId>
         </dependency>
-        <dependency>
-            <groupId>io.lighty.modules</groupId>
-            <artifactId>lighty-restconf-nb-community</artifactId>
```

Comment:
From 'example' application shouldn't be removed dependency for lighty-restconf-nb-community or any module directly started by example application.
This example takes dependency for lighty-restconf-nb-community transitively from swagger dependency. To resolve that currently dependency for lighty-restconf-nb-community is duplicate, I would add scope 'provided' inside swagger pom for lighty modules.